scripts/lint

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 # Summary: lint js files.
 
-npm run lint
+node_modules/.bin/eslint elements lib --max-warnings=1 --cache
```

Comment:
Why not run the npm script here? With this, we have a mix of commands and scripts in `package.json` file.